e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/view-utilisation.spec.js

```diff
@@ -133,16 +166,51 @@ context('Users can view utilisation', () => {
+      cy.assertText(pages.utilisationReportPage.utilisationTab.table.utilisation(), secondUtilisationString);
       cy.assertText(pages.utilisationReportPage.utilisationTab.table.coverPercentage(), '85%');
       // 85% of 199,999.99 is 169,999.9915
       cy.assertText(pages.utilisationReportPage.utilisationTab.table.exposure(), '169,999.99');
       cy.assertText(pages.utilisationReportPage.utilisationTab.table.feesAccrued(), `${CURRENCY.EUR} 200.00`);
       cy.assertText(pages.utilisationReportPage.utilisationTab.table.feesPayable(), `${CURRENCY.GBP} 100.00`);
     });
+
+    pages.utilisationReportPage.utilisationTab.table.row(FEE_RECORD_ID_THREE).within(() => {
+      cy.assertText(pages.utilisationReportPage.utilisationTab.table.facilityId(), FACILITY_ID_THREE);
+      cy.assertText(pages.utilisationReportPage.utilisationTab.table.exporter(), thirdExporter);
+      cy.assertText(pages.utilisationReportPage.utilisationTab.table.baseCurrency(), CURRENCY.GBP);
+      cy.assertText(pages.utilisationReportPage.utilisationTab.table.value(), '1,000,000.00');
+      cy.assertText(pages.utilisationReportPage.utilisationTab.table.utilisation(), thirdUtilisationString);
+      cy.assertText(pages.utilisationReportPage.utilisationTab.table.coverPercentage(), '80%');
+      // 85% of 90,000 is 72,000
+      cy.assertText(pages.utilisationReportPage.utilisationTab.table.exposure(), '72,000.00');
+      cy.assertText(pages.utilisationReportPage.utilisationTab.table.feesAccrued(), `${CURRENCY.EUR} 200.00`);
+      cy.assertText(pages.utilisationReportPage.utilisationTab.table.feesPayable(), `${CURRENCY.GBP} 100.00`);
+    });
+  });
+
+  describe('when sorting by ascending utilisation', () => {
+    it('should sort the reports correctly by utilisation', () => {
+      pages.utilisationReportPage.utilisationTab.table.utilisationHeader().click();
+
+      cy.assertText(pages.utilisationReportPage.utilisationTab.table.utilisation().eq(0), thirdUtilisationString);
+      cy.assertText(pages.utilisationReportPage.utilisationTab.table.utilisation().eq(1), secondUtilisationString);
+      cy.assertText(pages.utilisationReportPage.utilisationTab.table.utilisation().eq(2), firstUtilisationString);
+    });
+  });
+
+  describe('when sorting by descending utilisation', () => {
+    it('should sort the reports correctly by utilisation', () => {
+      // click twice for descending sort
+      pages.utilisationReportPage.utilisationTab.table.utilisationHeader().click();
+      pages.utilisationReportPage.utilisationTab.table.utilisationHeader().click();
+
+      cy.assertText(pages.utilisationReportPage.utilisationTab.table.utilisation().eq(0), firstUtilisationString);
+      cy.assertText(pages.utilisationReportPage.utilisationTab.table.utilisation().eq(1), secondUtilisationString);
+      cy.assertText(pages.utilisationReportPage.utilisationTab.table.utilisation().eq(2), thirdUtilisationString);
+    });
```

Comment:
Can we destructure in this file so it reads nicer, e.g

```ts
cy.assertText(table.someElement(), 'example');
```

Reply:
done